gitpod-io/website
# [Analytics] Track Form Submissions fully with Segment + Fix Do not track logic

## Description

PR introduces three improvements for Segment tracking:

1. Form submissions in `/community-license` are tracked
2. Submitted forms are fully tracked. This enables us to rely on the Segment events when processing forms submitted by users
3. The position of when we check for do not track signals is changed so that they are ignored for strictly necessary Segment calls. This ensures that the Segment event for forms are submitted even when users use Do not track.

## How to test


1. Submit forms on `/contact/sales`, `/contact/support`, and `/community-license` and ensure that the form submissions end up in the [Segment Debugger](https://app.segment.com/gitpod/sources/website_staging/debugger), including all the properties submitted
2. Turn on the do not track settings of your browser and repeat step 1)
